sites/public/pages/applications/start/choose-language.tsx

```diff
@@ -39,6 +39,10 @@ const ApplicationChooseLanguage = () => {
 
   const listingId = router.query.listingId
 
+  useEffect(() => {
+    conductor.reset()
+  }, [listingId])
+
```

Comment:
@YazeedLoonat , I think we determined that this would change each time, so we can include the reset at the top of the useEffect below here, no?